do/loadbalancers.go

```diff
@@ -226,10 +226,46 @@ func (l *loadbalancers) lbByName(ctx context.Context, name string) (*godo.LoadBa
 	return nil, lbNotFound
 }
 
+func (l *loadbalancers) dropletList(ctx context.Context) ([]godo.Droplet, error) {
```

Comment:
Since there is no AllDropletList at `godo` there are 3 choices.

Duplicating this code at LB and Droplet elements (which is current option at this PR)
Create an `instances` object at LB where needed (which would couple too much)
Create some common utilities at this package where the allDropletList would be hosted (not a fan of util.do or common.go kind of files here)

It's up to the maintainers to decide.